en.json

```diff
@@ -0,0 +1,573 @@
+{
+  "general": {
+    "meta": {
+      "tags": "tags \"{{ tags }}\"",
```

Comment:
not sure about the skip char here, please recheck if necessary

Reply:
Yes, we need the skip char for the quotes not to conflict with the JSON quotes.